recipes/b2btools/meta.yaml

```diff
@@ -1,62 +1,53 @@
-{% set name = "b2bTools" %}
-{% set version = "3.0.6" %}
+{% set name = "b2btools" %}
+{% set version = "3.0.7" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: 3617afcfe76cbff3a6bfad4060f962568565fb691b978f87d8db78cde97da58b
+  sha256: b3c9f10c10c8a12c390a79016e01b7e4d165bf1c94dce48e72d31891cb552c29
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
-  noarch: python
+  skip: true  # [py >= 311]
```

Comment:
I think this is the reason why nothing has been built.

Reply:
Thanks for your help @martin-g, really appreciated. We are trying to finish this recipe as soon as possible. 

Unfortunately there is a dependency (`pomegranate`) that doesn't support neither Py311 nor Py312 for the version we need to install `>=0.14.8, <0.14.10`. That limits our support to both Py311 and Py312 :/ The only way we found to install it using conda was requesting the dependency from `pip`, but that is not a possibility in bioconda recipes, isn't it ? Otherwise we could try to build that dependency from the sources, but I couldn't figure out how to make it. Would it be possible to do it in the build script ? Basically we would need to do:

```python
pip install "cython~=0.29.14" "scipy>=0.17.0"
git clone -b v0.14.9 --single-branch https://github.com/jmschrei/pomegranate.git
cd pomegranate
python setup.py install
```